HelpingHand/settings.py

```diff
@@ -52,6 +52,7 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'api.middleware.JsonMiddleware'
```

Comment:
just replace django.contrib.common.CommonMiddleware with this, since this one inherits from CommonMiddleware